base/metadata/metadata.go

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2024, Cogent Core. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package metadata
```

Comment:
You need a whole package for a simple map type??

Reply:
yes. metadata.Get must be at package scope! no generic methods.